manim/animation/animation.py

```diff
@@ -266,4 +277,4 @@ def update_mobjects(self, dt: float) -> None:
         pass
 
     def interpolate(self, alpha: float) -> None:
-        pass
+        pass
```

Comment:
Add a new line, run black.

Reply:
what does "Add a new line, run black." mean?

Reply:
I meant there should be a new line at the end of the file.

Running black will automatically do that.

Reply:
hm. strange. I have black running automatically in vscode but somehow it did not work this time... I have fixed it manually.